src/main/java/tds/content/mappers/ContentsItemDocumentMapper.java

```diff
@@ -267,15 +267,6 @@ private static APIPRelatedElementInfo getApipRelatedElementInfo(final RelatedEle
         return apipRelatedElementInfo;
     }
 
-    private static void processGenericElements(final ITSContent itsContent, final Content content) {
```

Comment:
This was removed as part of a bug we discovered when processing these "generic elements" - Jackson is having issues serializing/deserializing dom Elements properly, so we are removing this call for the time being. If we notice any issues during QA we can revisit.